apps/docs/docs/template/api/virtual-scrolling.md

```diff
@@ -0,0 +1,862 @@
+| `viewCacheSize`  | `number`                                                           | _default: `20`_ Controls the amount if views held in cache for later re-use when a user is scrolling the list If this is set to 0, `rxVirtualFor` won't cache any view, thus destroying & re-creating very often on scroll events.                                                                                                                  |
+
+### RxVirtualScrollViewportComponent
+
+Container component comparable to CdkVirtualScrollViewport acting as viewport for `*rxVirtualFor` to operate on.
+Its main purpose is to implement the `RxVirtualScrollViewport` interface as well as maintaining the scroll runways'
+height in order to give the provided `RxVirtualScrollStrategy` room to position items. Furthermore, it will gather and forward
+all events to the consumer of `rxVirtualFor`.
+
+#### Outputs
+
+| Output                | Type                                         | description                                                                                                                                                                                                                                                                                                                                                     |
+| --------------------- | -------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `viewRange`           | `ListRange: { start: number; end: number; }` | The range to be rendered by `*rxVirtualFor`. This value is determined by the provided `RxVirtualScrollStrategy`. It gives the user information about the range of items being actually rendered to the DOM. Note this value updates before the `renderCallback` kicks in, thus it is only in sync with the DOM when the next `renderCallback` emitted an event. |
+| `scrolledIndexChange` | `number`                                     | The index of the currently scrolled item. The scrolled item is the topmost item actually being visible to the user.                                                                                                                                                                                                                                             |
+
+### RxVirtualScrollStrategy
```

Comment:
Can we reference it in the fixed strategy section like "To see the instructions for other scrolling strategies see [this section]"